Src/HoneyBear.HalClient.Unit.Tests/HoneyBear.HalClient.Unit.Tests.xproj

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VisualStudioVersion Condition="'$(VisualStudioVersion)' == ''">14.0</VisualStudioVersion>
+    <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
+  </PropertyGroup>
+  <Import Project="$(VSToolsPath)\DotNet\Microsoft.DotNet.Props" Condition="'$(VSToolsPath)' != ''" />
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>317104fb-db9b-4ced-9d33-e0ddf67e8fa4</ProjectGuid>
+    <RootNamespace>HoneyBear.HalClient.Unit.TestsCore</RootNamespace>
```

Comment:
Could you rename the root namespace to `HoneyBear.HalClient.Unit.Tests`, please?  Just to keep things consistent.